src/Kdyby/Doctrine/EntityManager.php

```diff
@@ -74,7 +77,7 @@ public function bindTracyPanel(Diagnostics\Panel $panel)
 	/**
 	 * @return \Kdyby\Doctrine\QueryBuilder
 	 */
-	public function createQueryBuilder()
+	public function createQueryBuilder($alias = NULL, $indexBy = NULL)
```

Comment:
I don't like it either.

I was trying to go the way of least resistance, ofcourse they could be removed from the interface but I have no idea what that would do with existing code.